src/common/action-button.js

```diff
@@ -7,13 +7,16 @@ import { autobind, propTypes } from './utils'
 
 @propTypes({
   btnStyle: propTypes.string,
+  disabled: propTypes.bool,
+  form: propTypes.string,
   handler: propTypes.func.isRequired,
   handlerParam: propTypes.any,
   icon: propTypes.string.isRequired,
   size: propTypes.oneOf([
     'large',
     'small'
-  ])
+  ]),
+  type: propTypes.string
```

Comment:
Remove `type`, if there a form is provided, behave as a submit, otherwise as a button.